development/readme_eclipse_4.28.html

```diff
@@ -0,0 +1,1427 @@
+                <li><a href="#mozTocId73293">Importing in Eclipse application fails if plug-in exists in host workspace</a></li>
+                <li><a href="#mozTocId914239">Reusing a workspace after changing architectures silently breaks PDE models</a></li>
+                <li><a href="#mozTocId624312">Missing @since tag API Tools problems on interface fields containing @noreference tag</a></li>
+               </ol></li>
+                
+              </ol></li>
+          </ol></li>
+        <li><a href="#mozTocId19817">Running Eclipse</a>
+          <ol>
+            <li><a href="#mozTocId60052">Allocating enough memory and solving OutOfMemoryErrors</a></li>
+            <li><a href="#mozTocId735187">Selecting a workspace</a></li>
+            <li><a href="#mozTocId620725">Specifying the Java virtual machine</a></li>
+            <li><a href="#mozTocId928700">Mac OS X</a></li>
+            <li><a href="#mozTocId221199">Shared Install</a></li>
+          </ol></li>
+        <li><a href="#mozTocId575572">Upgrading Workspace from a Previous Release</a>
+          <ol>
+            <li><a href="#mozTocId401691">Users who don't use "-data"</a></li>
+            <li><a href="#mozTocId197750">Users who do use "-data"</a></li>
+            <li><a href="#mozTocId839433">Users who use User Libraries or classpath containers that contain JARs referencing other libraries via Class-Path in the MANIFEST.MF</a></li>
+            <li><a href="#mozTocId623237">Dropped in bundles may not resolve after upgrade</a></li>
+          </ol></li>
+        <li><a href="#mozTocId638978">Interoperability with Previous Releases</a>
+          <ol>
+            <li><a href="#mozTocId759237">Interoperability of Release 4.28 with previous releases</a>
+              <ol>
+                <li><a href="#mozTocId226004">Sharing projects between heterogeneous Eclipse 4.28 and 4.27</a></li>
+                <li><a href="#mozTocId574781">Using Eclipse 4.28 to develop plug-ins that work in Eclipse 4.27</a></li>
+              </ol>
+            </li>
+          </ol></li>
+  </ol>
+  <h2 id="TargetOperatingEnvironments2">
+    <a
+      class="mozTocH2"
+      name="mozTocId315234" />Target Operating Environments
+  </h2>
+  <p>In order to remain current, each Eclipse Project release targets reasonably current operating environments.</p>
+  <p>Most of the Eclipse SDK is "pure" Java code and has no direct dependence on the underlying operating system. The chief dependence is therefore on the Java Platform itself. Portions are
+    targeted to specific classes of operating environments, requiring their source code to only reference facilities available in particular class libraries (e.g. J2ME Foundation 1.1, J2SE 1.4, Java
+    5, etc).</p>
+  <p>In general, the 4.28 release of the Eclipse Project is developed on Java SE 11 VMs. As such, the Eclipse SDK as a whole is targeted at all modern, desktop Java VMs.</p>
```

Comment:
Minimum required version is Java 17 now.

Reply:
Updated